command/select.go

```diff
@@ -27,94 +28,238 @@ Options:
 	{{range .VisibleFlags}}{{.}}
 	{{end}}
 Examples:
-	01. Search for all JSON objects with the foo property set to 'bar' and spit them into stdout
-		 > s5cmd {{.HelpName}} --compression gzip --query "SELECT * FROM S3Object s WHERE s.foo='bar'" "s3://bucket/*"
+	01. Select the average price of the avocado and amount sold, set the output format as csv 
+		 > {{.HelpName}} --compression GZIP --output-format csv --query "SELECT s.avg_price, s.quantity FROM S3Object s WHERE s.item='avocado'" "s3://bucket/itemprices"
```

Comment:
This approach has a problem, where the `{{.HelpName}}` is the selected **subcommand**, so when I call `s5cmd select json --help`, it prints: 
```
  01. Select the average price of the avocado and amount sold, set the output format as csv
     > s5cmd select json --use-header USE --query "SELECT s.avg_price, s.quantity FROM S3Object s WHERE s.item='avocado'" "s3://bucket/prices.csv"
```
Which is wrong, since `json` subcommand does not take `--use-header` flag. That's the reason why examples are hardcoded.

Reply:
That's not my approach Deniz, this example was not working if the file is a CSV/TSV. If examples need to be hardcoded, then we should not use `.HelpName`. Please fix the example and merge this PR.